source/internals/options.js

```diff
@@ -90,8 +90,8 @@ const askiOSPods = () =>
 
 const askSystemiOSPodsCache = () =>
   new Promise(resolve => {
-    if (args.includes('--remove-system-iOS-pods-cache')) {
-      wipeSystemiOSPodsCache = true;
+    if (args.includes('--keep-system-iOS-pods-cache')) {
```

Comment:
Why the change from `remove` to `keep`?

Reply:
Sorry - it was in the commit message - I only noticed it yesterday as I was more fully integrating the new version into my workflow / doing release builds 

```
fix: flip pod cache removal arguments from "--remove" to "--keep"

They are in clean-project-auto and default to true per feedback on the feature request,
so then the previous argument style left no way to actually disable them by the command line
if you wanted to preserve your local pod caches

Now you can preserve them on the command line if you like
```

I *think* this mirrors the behavior of other tasks that are default-on and in auto mode, the flag in those is I think to keep things